src/__tests__/ParserComponent.test.tsx

```diff
@@ -436,6 +439,91 @@ describe('<ParserComponent /> Unit Tests', () => {
     });
   });
 
+  describe('lookForDateKey()', () => {
+    const wrapper = shallow(<ParserComponent prompt={'Select ' +
+    'a CSV file: '} fileType={FileType.csv}
+    onChange={function() {
+    }}/>);
+    const instance = wrapper.instance() as ParserComponent;
+    it('should return column name with the valid date in the first row' +
```

Comment:
Can you add tests that have the dates in other formats? So we can be sure it respects the `formatString`?